doc/changes/changes_15.1.0.md

```diff
@@ -0,0 +1,27 @@
+# Virtual Schema Common Java 15.1.0, released 2021-xx-xx
+
+Code name: 
+
+## Feature
+
+* 223: Added new scalar function capability: FN_TYPEOF
+
+## Documentation
+
+* #156: Improved documentation on how the virtual schema query processing works.
+## Dependency Updates
```

Comment:
add an empty line here. Unfortunately, PK can't do this until now...

Reply:
Fixed it yesterday but forgot to push